webapp/src/components/root/index.js

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2017-present Mattermost, Inc. All Rights Reserved.
+// See License for license information.
+
+const {connect} = window.ReactRedux;
+const {bindActionCreators} = window.Redux;
+import {getTheme} from 'mattermost-redux/selectors/entities/preferences';
+import {isRootModalVisible, getAuthenticated} from '../../selectors';
+import {startMeeting, closeRootModal, getOAuthConnectURL} from '../../actions';
+
+import Root from './root.jsx';
+
+function mapStateToProps(state, ownProps) {
+  const post = ownProps.post || {};
+
+  return {
+    visible: isRootModalVisible(state),
+    authenticated: getAuthenticated(state),
+    state,
```

Comment:
You are passing there the whole state, this is not how is expected to be used redux. Probably makes more sense to change it for something like `confg: getConfig(state)` and use the visible already passed prop instead of `this.props.state.rootModalVisible` used in the `Root` component.

Reply:
@jespino I've updated it to the existing `visible` in the props.  Is this more inline with what you were thinking? (I'm not well versed with redux, please forgive my ignorance)